judge/views/user.py

```diff
@@ -195,7 +203,7 @@ def get_context_data(self, **kwargs):
         context['submission_metadata'] = mark_safe(json.dumps({
             'min_year': (
                 self.object.submission_set
-                .annotate(year_only=ExtractYear('date'))
+                .annotate(year_only=ExtractYear('date', tzinfo=timezone.get_current_timezone()))
```

Comment:
This seems unnecessary — [the documentation](https://docs.djangoproject.com/en/2.2/ref/models/database-functions/#extract) states that the current timezone is automatically used, and the code confirms that just like above.

Reply:
Reverted.